src/nxt_process_title.c

```diff
@@ -178,6 +178,12 @@ nxt_process_arguments(nxt_task_t *task, char **orig_argv, char ***orig_envp)
     }
 
 done:
+    if (p != NULL) {
+       nxt_free(p);
```

Comment:
Hi,  I just saw this old PR around, and had a fast look at it.  nxt_free(), as well as free(3), can handle NULL just fine.  The conditionals are not necessary.